probe-rs/src/architecture/arm/dp/mod.rs

```diff
@@ -7,17 +7,18 @@ use jep106::JEP106Code;
 
 pub trait DebugPort {
     fn version(&self) -> &'static str;
```

Comment:
Wouldn't it be better if we changed this to return the version and then impl AsRef<str> for DebugPortVersion? or ToString for that ...

Reply:
I removed the Trait and version structs completely, they are not used right now.